en/docs/learn/configuring-inbound-provisioning-for-a-service-provider.md

```diff
@@ -1,6 +1,6 @@
 # Configuring Inbound Provisioning for a Service Provider
 
-See [Inbound Provisioning](../../learn/inbound-provisioning) for more information
+Click [here](../../learn/inbound-provisioning) for more information
```

Comment:
when linking to other pages in the doc space, we generally follow the format "See xxx for more information" or "For information, see xxx".

Is there a reason to change this?

Reply:
See Inbound Provisioning for more information on inbound provisioning.
is it ok to document like this two Inbound Provisioning ?

Reply:
Ohh... Instead, shall we remove this line and just link the word "inbound provisioning" in the next sentence (To configure inbound provisioning for the service provider.....) to that page?